### PR TITLE
[MIG] [17.0] import_export_templating: migrated module.

### PR DESCRIPTION
**Pull request by ShwetaSerpentCS**
Responsible : Utsav

### CODE REVIEW

**Comment by ketanserpentcs** [comment]
Testing is assigned to Pooja

File: import_export_templating/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    import-export for non-technical users of ERP.
+    """,
+    "depends": ["web"],
+    "data": ["security/ir.model.access.csv", "wizard/wiz_download_template_view.xml"],
+    # Odoo App Store Specific
+    "images": ["static/description/Banner_import_export_templating_17.png"],
+    "installable": True,
+    "price": 30.0,
+    "currency": "EUR",
+    # "assets": {
```
Comment (Trivedi-Vacha-SerpentCS) [inline import_export_templating/__manifest__.py]: why commented the code?

File: import_export_templating/wizard/wiz_download_template.py [hunk 2]
```diff
@@ -0,0 +1,625 @@
...
+        "ir.model.fields", compute="_compute_get_names"
+    )
+    fields_list_ids = fields.Many2many(
+        "ir.model.fields", domain="[('model_id', '=', ir_model)]"
+    )
+
+    @api.onchange("type")
+    def _get_active_model(self):
+        if self._context and self._context.get("active_model"):
+            model = self._context.get("active_model")
```
Comment (Trivedi-Vacha-SerpentCS) [inline import_export_templating/wizard/wiz_download_template.py:45]: `model = self._context.get("active_model") 

if model:`

**Review by Trivedi-Vacha-SerpentCS — CHANGES_REQUESTED** [review]
@ShwetaSerpentCS I can see scope of code optimisation, can assign to someone familiar with excel report

**Comment by ShwetaSerpentCS** [comment]
> @ShwetaSerpentCS I can see scope of code optimisation, can assign to someone familiar with excel report

I'm assigning to Ankit vaja.  @Trivedi-Vacha-SerpentCS

**Comment by ketanserpentcs** [comment]
Testing has been done with index by Pooja

**Review by Trivedi-Vacha-SerpentCS — APPROVED** [review]

**Pull request merged**